upload.py

```diff
@@ -72,15 +72,16 @@ def ntransfercmd(self, cmd, rest=None):
 print(">>> Uploading .bw files to FTP server.")
 
 # Set up FTPS connection
-ftps = FTPS_connect (host='129.114.60.56', user=args.username, passwd=password)
+ftps = FTPS_connect (host='usegalaxy.org', user=args.username, passwd=password)
 ftps.set_debuglevel(10)
 ftps.prot_p()
 ftps.retrlines('LIST')
 
 # Upload all .bw filenames to the galaxy FTP server.
 for filename in (glob.glob(args.dir + "/*.bw")):
     f = open(filename, 'rb')
-    cmd = 'STOR '+filename
+    bwname = filename.split("/")[-1]
```

Comment:
`os.path.basename(path)` is better than manually splitting and extracting the filename.